app/controllers/graphql_controller.rb

```diff
@@ -13,6 +13,10 @@ def execute
       # current_user: current_user,
     }
     result = NextGenAuthAppDemoSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+    if result.to_h['data']['testField']['errors'].present?
```

Comment:
@ragalie This doesn't seem like the best way to check whether we get 401s from the graphql request. Currently, graphql is wrapping the error as a 200 response. Do you know if there is a better pattern for this?

Reply:
I think there are a couple types of GraphQL errors. This might be a good fit for the built-in `errors` since it's developer-facing: https://development.shopify.io/engineering/developing_at_Shopify/apis/building/graphql/errors